daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -359,6 +359,53 @@ function Set-Repos {
   Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' addrepo 'google-compute-engine-stable' 'https://packages.cloud.google.com/yuck/repos/google-compute-engine-stable'
 }
 
+function Export-ImageMetadata {
+  $id =  (Get-WmiObject -Class Win32_ComputerSystemProduct).UUID.ToLower()
+  $edition = Get-MetadataValue -key 'edition'
```

Comment:
Don't pull these from metadata, they should be pulled from the OS itself.

Reply:
Fix in next commit